tests/measureSources.py

```diff
@@ -237,15 +237,21 @@ def testPixelFlags(self):
         edge = mask.getPlaneBitMask('EDGE')
         bad = mask.getPlaneBitMask('BAD')
         nodata = mask.getPlaneBitMask('NO_DATA')
+        # Add in the non default mask plane CLIPPED (DM-2915)
```

Comment:
No need to sprinkle the ticket number through the comments -- it's easy enough to find with `git blame`.